napari/_qt/menus/_tests/test_file_menu.py

```diff
@@ -44,33 +45,34 @@ def _(path):
 
 
 def test_plugin_display_name_use_for_multiple_samples(
-    make_napari_viewer, builtins
+    make_napari_viewer,
+    builtins,
 ):
-    """For plugin with more than two sample datasets, should use plugin_display for building the menu"""
+    """Check 'display_name' used for submenu when plugin has >1 sample data."""
+    app = get_app()
+    # Note this is required so `init_qactions` is run
     viewer = make_napari_viewer()
```

Comment:
@DragaDoncila I think we questioned whether the sample data actions should be qactions, since we would like to be able to open sample data without a viewer. With the sample data commands being qactions atm, is there a way to run the open sample callback without a viewer, since the actions are only registered after `init_qactions`, which is done in `_QtMainWindow`?

Reply:
Nevermind, I did not realise headless has access to:

https://github.com/napari/napari/blob/559eb525194267bb4ebb38e2987006f8883f142e/napari/components/viewer_model.py#L666-L672